ballerina/tests/test.bal

```diff
@@ -31,41 +30,57 @@ ConnectionConfig config = {
     awsCredentials: awsCredentials
 };
 
-Client amazonSimpleDBClient = check new(config);
+Client amazonSimpleDBClient = check new (config);
 
-@test:Config{}
+@test:Config {}
 function testCreateDomain() returns error? {
     CreateDomainResponse|xml response = check amazonSimpleDBClient->createDomain("test");
-    log:printInfo(response.toString());
+    if response is xml {
+        assert(response);
+    }
 }
 
-@test:Config{dependsOn: [testCreateDomain]}
+@test:Config {dependsOn: [testCreateDomain]}
 function testListDomains() returns error? {
     ListDomainsResponse|xml response = check amazonSimpleDBClient->listDomains();
-    log:printInfo(response.toString());
+    if response is xml {
+        assert(response);
+    }
 }
 
-@test:Config{dependsOn: [testListDomains]}
+@test:Config {dependsOn: [testListDomains]}
 function testGetDomainMetaData() returns error? {
     DomainMetaDataResponse|xml response = check amazonSimpleDBClient->getDomainMetaData("test");
-    log:printInfo(response.toString());
+    if response is xml {
+        assert(response);
+    }
 }
 
-@test:Config{dependsOn: [testGetDomainMetaData]}
+@test:Config {dependsOn: [testGetDomainMetaData]}
 function testSelect() returns error? {
-    string selectExpression = "select output_list from test"; 
+    string selectExpression = "select output_list from test";
     SelectResponse|xml response = check amazonSimpleDBClient->'select(selectExpression, true);
-    log:printInfo(response.toString());
+    if response is xml {
+        assert(response);
+    }
 }
 
-@test:Config{dependsOn: [testSelect]}
+@test:Config {dependsOn: [testGetAttributes]}
 function testDeleteDomain() returns error? {
     DeleteDomainResponse|xml response = check amazonSimpleDBClient->deleteDomain("test");
-    log:printInfo(response.toString());
+    if response is xml {
+        assert(response);
+    }
```

Comment:
IMO we can move these type checks into the `assert()` util itself, to avoid having duplications. WDYT?
```suggestion
    if response is xml {
        assert(response);
    }
```